tests/OpenTK.Tests.Math/QuaternionTests.cs

```diff
@@ -7,121 +7,152 @@ namespace OpenTK.Tests.Math
 	public class Quaternion_Tests
 	{
 		/// <summary>
-		/// Checks if a single given value (either pitch, yaw or roll) get converted into correct x,y,z value of quaternion.
+		/// Contains all tests which cover the ctor of Quaternion.
 		/// </summary>
-		/// <param name="eulerValues">euler angle values</param>
-		/// <param name="expectedResult">expected xyz component of quaternion</param>
-		/// <param name="testName">Taken from nUnit test data. Don't know how to name data driven tests for xUnit which actually works.</param>
-		[Theory]
-		[MemberData(nameof(QuaternionTestDataGenerator.SingleAxisTestCases), MemberType = typeof(QuaternionTestDataGenerator))]
-		public void CtorEulerAnglesFloat_SingleEulerAngleSet_RotateCorrectAxis(Vector3 eulerValues, Vector3 expectedResult, string testName)
+		public class Constructor_Tests
```

Comment:
You can name this "Constructor". We know they're tests.

Reply:
Done.